dex/testing/eth/harness.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/sh
+# Tmux script that sets up a simnet harness.
+set -ex
+
+SESSION="eth-harness"
+
+CHAIN_ADDRESS_JSON_FILE_NAME="UTC--2021-01-27T08-20-38.123221057Z--9ebba10a6136607688ca4f27fab70e23938cd027"
+CHAIN_ADDRESS="9ebba10a6136607688ca4f27fab70e23938cd027"
+CHAIN_ADDRESS_JSON='{"address":"9ebba10a6136607688ca4f27fab70e23938cd027","crypto":{"cipher":"aes-128-ctr","ciphertext":"dcfbe17de6f315c732855111b782496d76b2d703169afddaaa69e1bc9e02ec51","cipherparams":{"iv":"907e5e050649d1c5c0be782ec7db5cf1"},"kdf":"scrypt","kdfparams":{"dklen":32,"n":262144,"p":1,"r":8,"salt":"060f4e16d601069a6bccae0693a15cd72090baf1ab20e408c89883117d4f7c51"},"mac":"b9ca7dad75a04b77dc7751a814c051f32752603334e4bb4046caf927196a5579"},"id":"74805e39-6a2f-46eb-8125-70c41d12c6d9","version":3}'
```

Comment:
This is basically what an eth wallet looks like. The wiki says that users should use this: https://geth.ethereum.org/docs/clef/tutorial
afaict, clef can provide deterministic keys. These are then saved as json here in the datadir/keystore directory. As I picture it now, we will need to require a user's clef master key and clef password and we will have to do all of the address management ourselves. I am of course an eth noob so not sure at all yet.

Reply:
Adding clef "wallets" in another pr. Perhaps they cannot be used while mining? I was able to cause some panics... Will add separate trading node/wallets that make use of clef.

Reply:
I've decided against using clef. It only seems to complicate things. We can get what we need straight from geth. I don't think geth's account management should be something we should have to worry about so much. I have the basics for using clef going here, in case it turns out we should use it: https://github.com/JoeGruffins/dcrdex/commit/2240d499c5c938f121ff9351f6d261a936ecacbc

From the clef [readme](https://github.com/ethereum/go-ethereum/tree/master/cmd/clef) it looks like this is mostly a plus for remote signers. Since we will be requiring the eth node be local, I don't think it benefits us. If we decided to go with explorers, we would use it. That has not been our design pattern so far however.

Reply:
How did you generate this json for `CHAIN_ADDRESS_JSON`?

Reply:
I just copy pasted straight from an "account" created with geth. You can find them as files in `~/.ethereum/keystore/long-name-with-date`.